### PR TITLE
Adding the source files from git so the master has the examples.

### PR DESCRIPTION
**Pull request by mbruzek**
Had to change the variables to install the kubernetes source code.

### CODE REVIEW

File: hooks/hooks.py [hunk 1]
```diff
@@ -32,8 +32,7 @@ def config_changed():
     CHARM_DIR = os.environ.get('CHARM_DIR', '')
     kubernetes_file = os.path.join(CHARM_DIR, 'files', kubernetes_tar_file)
     installer = KubernetesInstaller(arch, version, kubernetes_file)
-    # Install the kubernetes binary files in the /opt/kubernetes/bin directory.
-    installer.install('/opt/kubernetes/bin')
+    installer.install('/opt/kubernetes')
```
Comment (lazypower) [inline hooks/hooks.py:36]: THumbs up on removing the comment - can be leave what the installer routine is doing here though?

File: hooks/kubernetes_installer.py [hunk 2]
```diff
@@ -22,12 +22,18 @@ def __init__(self, arch, version, kubernetes_file):
         self.version = version
         self.kubernetes_file = kubernetes_file
 
-    def install(self, output_dir='/opt/kubernetes/bin'):
+    def install(self, output_dir='/opt/kubernetes'):
         """ Install kubernetes binary files from the tar file or gsutil. """
```
Comment (lazypower) [inline hooks/kubernetes_installer.py:26]: Does this still install from gsutil now that we are installing from git though?

**Comment by lazypower** [comment]
@mbruzek Couple of minor knitpicks about comments, otherwise :+1: LGTM - i haven't run the code in question however.

File: hooks/kubernetes_installer.py (hunk 2 above)
Comment (mbruzek) [inline hooks/kubernetes_installer.py:26]: Yes the change away from gsutil is a bigger change.  The code still uses gsutil to get the binary files, and we use get to download the examples.  At this moment.

File: hooks/hooks.py (hunk 1 above)
Comment (mbruzek) [inline hooks/hooks.py:36]: Thanks for the review.  I will add back in the comment describing what the install method does.

**Comment by mbruzek** [comment]
I addressed your review comments and am going to merge this code so I can use the kubernetes source.

**Pull request merged**